service/history/shard/context_impl.go

```diff
@@ -602,24 +606,35 @@ func (s *ContextImpl) UpdateNamespaceNotificationVersion(namespaceNotificationVe
 	return nil
 }
 
-func (s *ContextImpl) UpdateHandoverNamespaces(namespaces []*namespace.Namespace, maxRepTaskID int64) {
+func (s *ContextImpl) UpdateHandoverNamespaces(namespaces []*namespace.Namespace) {
 	s.wLock()
 	defer s.wUnlock()
 
-	newHandoverNamespaces := make(map[string]struct{})
+	maxReplicationTaskID := s.immediateTaskExclusiveMaxReadLevel
```

Comment:
When there is no replication task (in low traffic namespace case), will the replication ack level for remote cluster move forward?

Reply:
Yes, even when there's no task at all, the ack level for replication queue will still keep moving forward. https://github.com/temporalio/temporal/blob/master/service/history/replication/ack_manager.go#L159 

I also double check the logic for making this happen when writing this PR and it looks good to me.